src/usr/local/www/system_usermanager_settings.php

```diff
@@ -134,7 +131,7 @@
 			}
 		}
 
-		if (isset($_POST['session_timeout']) && $_POST['session_timeout'] != "") {
+		if (is_numericint($_POST['session_timeout'])) {
```

Comment:
Note - we don't need to test further at this point, because if it isn't either a positive integer or (empty string/unset), $input_errors[] will have been nset above and we'll never get to this line anyway